internal/gatewayapi/route.go

```diff
@@ -444,6 +437,42 @@ func (t *Translator) processGRPCRouteRule(grpcRoute *GRPCRouteContext, ruleIdx i
 	return ruleRoutes
 }
 
+func (t *Translator) processGRPCRouteMethodExact(method *v1alpha2.GRPCMethodMatch, irRoute *ir.HTTPRoute) {
+	switch {
```

Comment:
I know you're just moving code here, but not gonna lie: this feels like a more natural candidate for `if` than `switch` -- it took me a minute to sort out what was up since there's no default case in the switch... 🙂

Reply:
after thinking about it, `if` could never be better, thanks.

Reply:
the lint suggests `switch` stmt, so I kept it

Reply:
Silly linter. 😂